modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -14,10 +14,11 @@ In the table, features are marked with the following statuses:
 |===
 |*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
 
-|*Version*|*kam*    |*Helm*  |*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*     |*RH SSO* |
-|1.5.0    |0.0.42 TP|3.8.0 GA|4.4.1 GA   |2.3.3 GA |0.4.1 TP        |2.30.3 GA |7.5.1 GA |4.8-4.10
-|1.4.0    |0.0.41 TP|3.7.1 GA|4.2.0 GA   |2.2.2 GA |0.2.0 TP        |2.30.0 GA |7.4.0 GA |4.7-4.9
-|1.3.0    |0.0.40 TP|3.6.0 GA|4.2.0 GA   |2.1.2 GA |0.2.0 TP        |2.28.0 GA |7.4.0 GA |4.7-4.9
+|*Version*|*kam*    |*Helm*  |*Kustomize*|*Argo CD*|*Dex*     |*RH SSO* |
+|1.6.0    |0.0.46 TP|3.8.1 GA|4.4.1 GA   |2.4.5 GA |2.30.3 GA |7.5.1 GA |4.8-4.10
+|1.5.0    |0.0.42 TP|3.8.0 GA|4.4.1 GA   |2.3.3 GA |2.30.3 GA |7.5.1 GA |4.8-4.10
+|1.4.0    |0.0.41 TP|3.7.1 GA|4.2.0 GA   |2.2.2 GA |2.30.0 GA |7.4.0 GA |4.7-4.9
+|1.3.0    |0.0.40 TP|3.6.0 GA|4.2.0 GA   |2.1.2 GA |2.28.0 GA |7.4.0 GA |4.7-4.9
 
 |===
 
```

Comment:
The table seems to be out of alignment.  Also, should be keep the `ApplicationSet` column?  We may want to display the ApplicationSet versions of the other OpenShift GitOps releases.
![Screenshot from 2022-07-22 08-29-45](https://user-images.githubusercontent.com/44041377/180439783-8b143da9-76f2-447c-aa82-c3832ef2a08e.png)